market_nextdoor_api/views.py

```diff
@@ -23,10 +23,13 @@ def get_customer_list(request):
   return Response(serializer.data)
 
 def create_customer(request):
+  #Similar to creating an user in rails = 
   serializer = CustomerSerializer(data=request.data)
   if serializer.is_valid():
     serializer.save()
     return Response(serializer.data, status=status.HTTP_201_CREATED)
+  else:
+    return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
This looks great, good catch on the customer creation error. Guessing we need to add the same errors to the rest of the creation methods?